src/yogiraj/Assignment20.java

```diff
@@ -0,0 +1,23 @@
+package yogiraj;
+
+public class Assignment20 {
+	int sum = 0;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.